test-it/karma/javascriptinvoke/DataTypes.Test.js

```diff
@@ -232,62 +284,4 @@ describe('A JavaScript function invoke', function () {
             done();
         });
     });
-
-    it('successfully create and use an object type', function (done) {
```

Comment:
This test exercising a lot more than JavaScript Reference parameter passing behavior like the other functions in this file. Moved to separate DiagramUsage test.